l10n/en/firefox/more.ftl

```diff
@@ -22,6 +22,14 @@ get-the-privacy = Get the privacy you deserve. Enhanced Tracking Protection make
 take-your-passwords = Take your passwords everywhere
 securely-access-the = Securely access the passwords you’ve saved in { -brand-name-firefox } from anywhere — even outside of the browser.
 choose-which-firefox = Choose which { -brand-name-firefox-browser } to download in your language
+firefox-chromebook = { -brand-name-firefox-browser } for Chromebook
```

Comment:
I know it's too late to add the namespace to the existing strings but we could start now with the new ones.